sdk/containerregistry/container-registry/test/public/containerRegistryClient.spec.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import { Context } from "mocha";
+
+import { ContainerRegistryClient } from "../../src";
+
+import { ClientSecretCredential } from "@azure/identity";
+import { env, record, Recorder } from "@azure/test-utils-recorder";
+
+// When the recorder observes the values of these environment variables in any
+// recorded HTTP request or response, it will replace them with the values they
+// are mapped to below.
+const replaceableVariables: Record<string, string> = {
+  CONTAINER_REGISTRY_ENDPOINT: "https://myappconfig.azconfig.io",
```

Comment:
Nit: Should we make this look like a container registry endpoint?

Reply:
Updated